Source/SDK/Client.cs

```diff
@@ -32,6 +36,11 @@ public class Client
         /// </summary>
         public IExecutionContextManager ExecutionContextManager { get; }
 
+        /// <summary>
+        /// Gets the <see cref="EventTypes" />.
+        /// </summary>
+        public EventTypes Artifacts { get; }
```

Comment:
```suggestion
        /// <summary>
        /// Gets the <see cref="EventTypes" />.
        /// </summary>
        public EventTypes EventTypes { get; }
```

This property should probably get renamed to EventTypes.